docs/installation/podman/Deploying_PKI_ACME_Responder_on_Podman.md

```diff
@@ -119,6 +123,31 @@ $ podman exec -ti pki-acme \
     psql postgres://acme:Secret.123@localhost.localdomain/acme
 ```
 
+## Deploying Permanent Realm
+
+Prepare a folder (e.g. realm) and store the configuration parameters in separate files.
+For example:
+
+- **class**: org.dogtagpki.acme.realm.PostgreSQLRealm
```

Comment:
Is existing QE automation also need to update or fail with existing config?
Currently it is using class PostgreSQLDatabase.
**class: org.dogtagpki.acme.database.PostgreSQLDatabase**

Reply:
I think we should update the test to use `PostgreSQLRealm` in the realm configuration. There might have been a bug in the old code that used `PostgreSQLDatabase` but it's been fixed.

I think most people will use the CLI or the example to configure ACME instead of writing from scratch so the chance to have a wrong class is low. We can certainly do a negative test by specifying a wrong class, but that's a low priority. Let's make sure all positive scenarios are covered first.